keyboards/xd96/keymaps/uuupah/keymap.c

```diff
@@ -0,0 +1,78 @@
+	  KC_LCTL, KC_A,    KC_S,    KC_D,    KC_F,    KC_G,    KC_H,    KC_J,    KC_K,    KC_L,    KC_SCLN, KC_QUOT, LT_ENT2,                   KC_P4,   KC_P5,   KC_P6,   KC_PPLS,
+	  KC_LSFT, KC_BSLS, KC_Z,    KC_X,    KC_C,    KC_V,    KC_B,    KC_N,    KC_M,    KC_COMM, KC_DOT,  KC_SLSH, KC_RSFT, MO(1),            KC_P1,   KC_P2,   KC_P3,   EQUALS,
+	  KC_LCTL, KC_LGUI, KC_LALT,                            KC_SPC,                             KC_RALT, KC_RGUI, KC_APP,  KC_RCTL,          KC_P0,   ZEROZER, KC_PDOT, KC_PENT
+  ),
+
+  [1] = LAYOUT_all(
+    _______, _______, _______, _______, _______, _______, _______, _______, _______, _______, _______, _______, _______, RGB_TOG, RESET,   _______, _______, _______, _______,
+	  _______, _______, _______, _______, _______, _______, _______, _______, _______, _______, _______, _______, _______, _______, _______, _______, _______, _______, _______,
+	  _______, KC_MPLY, KC_VOLU, _______, _______, _______, _______, _______, _______, _______, _______, KC_UP,   KC_PSCR, _______,          _______, _______, _______, _______,
+	  _______, KC_MPRV, KC_MNXT, _______, _______, _______, _______, _______, _______, _______, KC_LEFT, KC_RGHT, _______,                   _______, _______, _______, _______,
+	  _______, _______, KC_VOLD, _______, _______, _______, _______, _______, _______, _______, _______, KC_DOWN, _______, _______,          _______, _______, _______, _______,
+	  _______, _______, _______,                            _______,                            _______, _______, _______, _______,          _______, _______, _______, _______
+  ),
+  
+  [2] = LAYOUT_all(
+    _______, _______, _______, _______, _______, _______, _______, _______, _______, _______, _______, _______, _______, _______, _______, _______, _______, _______, _______,
+	  _______, _______, _______, _______, _______, _______, _______, _______, _______, _______, _______, _______, _______, _______, _______, _______, _______, _______, _______,
+	  _______, _______, _______, _______, _______, _______, _______, _______, _______, _______, _______, KC_PGUP, _______, _______,          _______, _______, _______, _______,
+	  _______, _______, _______, _______, _______, _______, _______, _______, _______, _______, KC_HOME, KC_END,  _______,                   _______, _______, _______, _______,
+	  _______, _______, _______, _______, _______, _______, _______, _______, _______, _______, _______, KC_PGDN, _______, _______,          _______, _______, _______, _______,
+	  _______, _______, _______,                            _______,                            _______, _______, _______, _______,          _______, _______, _______, _______
+  ),
+};
+
+bool process_record_user(uint16_t keycode, keyrecord_t *record) {
+  switch (keycode) {
+    case ZEROZER:
+      if (record->event.pressed) {
+        SEND_STRING("00");
+      } else {
+      }
+      break;
+    case EQUALS:
```

Comment:
Why not use `KC_EQL`?

Reply:
the initial intention was to emulate the realforce 23u equals key behaviour, but it seems that the string version acts the same as kc_eql so i've just reverted it